### ISSUE TITLE
[3.3.5] Orbaz Bloodbane (31283)

### ISSUE THREAD

**Issue by Treeston** [TrinityCore/TrinityCore#16168]
Branch 3.3.5, rev. 8a0bbc386982
[Quest: The Fate of Bloodbane](http://www.wowhead.com/quest=13164/the-fate-of-bloodbane)
[Creature: Orbaz Bloodbane](http://www.wowhead.com/npc=31283)

Bloodbane uses a single, unrelated line for all but the first of his chats. The line used is `broadcast_text` id `31259`: `There's no doubt that something has distorted the true history of Stormhoof and the warriors of Camp Tunka'lo. But, what?`

**Comment by tkrokli** [comment]
Any idea what it actually should look like from live servers? I would not mind looking into it if I had the info, but I don't have an online account (trial is no use here).

**Comment by dr-j** [comment]
Hmm Deja Vu

~~~sql
UPDATE `creature_text` SET `BroadcastTextId`=31961 WHERE  `entry`=31283 AND `groupid`=1 AND `id`=0;
UPDATE `creature_text` SET `BroadcastTextId`=31962 WHERE  `entry`=31283 AND `groupid`=2 AND `id`=0;
UPDATE `creature_text` SET `BroadcastTextId`=31963 WHERE  `entry`=31283 AND `groupid`=3 AND `id`=0;
UPDATE `creature_text` SET `BroadcastTextId`=31966 WHERE  `entry`=31283 AND `groupid`=4 AND `id`=0;
UPDATE `creature_text` SET `BroadcastTextId`=31967 WHERE  `entry`=31283 AND `groupid`=5 AND `id`=0;
UPDATE `creature_text` SET `BroadcastTextId`=31964 WHERE  `entry`=31283 AND `groupid`=6 AND `id`=0;

**Comment by dr-j** [comment]
and since I fixed that last february the below probably has not been pushed to tc either which was some updates to my earlier script for Betrayal.

~~~SQL
DELETE FROM `smart_scripts` WHERE `entryorguid`=28998 AND `source_type`=0 AND `id` in(13,14);UPDATE `world3`.`smart_scripts` SET `event_param2`=7500, `event_param3`=10000, `event_param4`=20000 WHERE  `entryorguid`=28998 AND `source_type`=0 AND `id`=5 AND `link`=0;
UPDATE `smart_scripts` SET `event_param2`=7500, `event_param3`=10000, `event_param4`=20000 WHERE  `entryorguid`=28998 AND `source_type`=0 AND `id`=5 AND `link`=0;

INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
(28998, 0, 13, 0, 0, 0, 100, 0, 0, 0, 60000, 60000, 28, 54104, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Overlord Drakuru - IC - Remove Aura (Blight Fog'),
(28998, 0, 14, 0, 0, 0, 100, 0, 45000, 45000, 60000, 60000, 11, 54104, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Overlord Drakuru - IC - Cast Blight Fog');

**Comment by tkrokli** [comment]
Nice, thanks for posting a solution so quickly. :+1: What was the "déjà vu" you had there? :-)

**Comment by dr-j** [comment]
I had fixed this last february p: just had failed to commit to tc -:)

**Comment by tkrokli** [comment]
Great, thanks for the extra info. :-) Keep up the good work, I will do what I can on my part.

**Comment by Treeston** [comment]
Closed, 3af1d91 and 19462ef.
